src/main/java/ch/epfl/biop/imaris/EasyXT.java

```diff
@@ -1241,6 +1243,66 @@ public static ISurfacesPrx create(ImagePlus imp, int timepoint) throws Error {
             return surface;
         }
 
+        /**
+         * create a new surfaces object from a Label ImagePlus
+         *
+         * @param impLabel the image to get a Surfaces from.
+         *                 A label image, each label will be a surface of the Surfaces object.
+         * @return the ISurfacesPrx with individual surface for each label value
+         * @throws Error an Imaris Error if there was a problem
+         */
+        public static ISurfacesPrx createFromLabels(ImagePlus impLabel) throws Error {
```

Comment:
Why is this function not making a loop for all timepoints ? I'm not saying it should do it, but maybe explain at least in the javadoc why it's not the case (when there's no timepoint argument, in the rest of the EasyXT, usually it means that all timepoints are processed). Is it an option to remove this function and just let the one with a timepoint argument ?

Reply:
I followed example made by @lacan for the function create()

Reply:
My guess is, this a lazy function, without time argument you start at 0 (or you added time tag to the image). Anyway, it calls the function with defined time-point that does the job for the time-points...

Reply:
Ah ok I get it, I thought it was making the job only for one timepoint, but it's just an offset. Then maybe just rename the variable to "imarisTimepointOffset" or "timepointOffset" ?

Reply:
I'd keep it consistent for the functions _create_ and _createFromLabels_, and let @lacan decide on it.
None of the other function mentions **offset** in the variable name, addChannel has zStart, tStart...

Reply:
Then tStart ?